client/components/indeterminate-checkbox/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from 'react';
+import Gridicon from 'components/gridicon';
+
+const IndeterminateCheckbox = ( { selectedCount, allCount, onToggle } ) => {
+	const allSelected = selectedCount === allCount;
+	const someSelected = 0 < selectedCount && allCount > selectedCount;
+
+	const onClick = ( event ) => {
+		const checked = ! ( someSelected || allSelected );
```

Comment:
Good point about consistent behaviour. I still think this would be better in the reducers though, having a trail of `checked` parameters passing through half the app just seems pointless. What about we let the next reviewer break the tie? I don't have any more comments about this PR and I don't want to hold it for a small thing like this.